src/providers/completion/styleCompletionItemProvider.ts

```diff
@@ -50,7 +50,7 @@ export class StyleCompletionItemProvider implements CompletionItemProvider {
 		} else if (/^['"][.#]\w*$/.test(linePrefix)) {
 			return this.getClassOrIdCompletions(linePrefix, prefix);
 			// tag - "Wind_ or "_
-		} else if (/^['"]\w*$/.test(linePrefix)) {
+		} else if (/^['"]\w['"]*$/.test(linePrefix)) {
```

Comment:
I think this regex change needs to be mirrored to the other checks as I'm not getting suggestions based off classes/ids from the view files